lib/licensed/git.rb

```diff
@@ -9,7 +9,7 @@ def available?
 
       def git_repo?
         return false unless available?
-        !Licensed::Shell.execute("git", "status", allow_failure: true).empty?
+        !Licensed::Shell.execute("git", "rev-parse", "HEAD", allow_failure: true).empty?
```

Comment:
🤔 `repository_root` already uses `git rev-parse --show-toplevel`, is that heavier weight than `git rev-parse HEAD`?  maybe this could be reworked like

```
def repository_root
  return unless available?
  Licensed::Shell.execute("git", "rev-parse", "--show-toplevel", allow_failure: true)
end

def git_repo?
  repository_root.empty?
end
```

Reply:
@jonabc That might be possible yeah, you're probably the better expert here than me. If that works as well, might be a good further simplification.